topics/admin/tutorials/ansible-galaxy/group_vars/galaxyservers.yml

```diff
@@ -38,6 +38,26 @@ galaxy_config:
     check_migrate_tools: false
     tool_data_path: "{{ galaxy_mutable_data_dir }}/tool-data"
     job_config_file: "{{ galaxy_config_dir }}/job_conf.xml"
+    # SQL Performance
+    database_engine_option_server_side_cursors: true
+    slow_query_log_threshold: 5
+    enable_per_request_sql_debugging: true
+    # File serving Performance
+    nginx_x_accel_redirect_base: /_x_accel_redirect
+    # Automation / Ease of Use
+    watch_job_rules: 'auto'
+    allow_path_paste: true
+    enable_quotas: true
+    allow_user_deletion: true
+    show_welcome_with_login: true
+    expose_user_name: true
+    expose_dataset_path: true
+    expose_potentially_sensitive_job_metrics: true
+    # Debugging
+    cleanup_job: onerror
+    allow_user_impersonation: true
+    # Tool security
+    outputs_to_working_directory: true
```

Comment:
Another option that I always have to add is:

```yaml
    retry_job_output_collection: 3
```

Not sure if it makes sense for GAT though.

Reply:
No, I like it, let's include it.